libs/audit-service/src/main/java/uk/gov/di/ipv/core/library/auditing/extension/AuditExtensionsVcEvidence.java

```diff
@@ -44,32 +42,21 @@ public class AuditExtensionsVcEvidence implements AuditExtensions {
     private final Integer age;
 
     public AuditExtensionsVcEvidence(
-            @JsonProperty(value = "iss", required = false) String iss,
-            @JsonProperty(value = "evidence", required = false) String evidence,
-            @JsonProperty(value = "successful", required = false) Boolean successful,
-            @JsonProperty(value = "vot", required = false) Vot vot,
-            @JsonProperty(value = "isUkIssued", required = false) Boolean isUkIssued,
-            @JsonProperty(value = "age", required = false) Integer age)
-            throws AuditExtensionException {
-        try {
-            this.iss = iss;
-            this.evidence = evidence == null ? null : OBJECT_MAPPER.readTree(evidence);
-            this.successful = successful;
-            this.vot = vot;
-            this.isUkIssued = isUkIssued;
-            this.age = age;
-        } catch (JsonProcessingException e) {
-            throw new AuditExtensionException(e);
-        }
+            @JsonProperty(value = "iss") String iss,
+            @JsonProperty(value = "evidence") JsonNode evidence,
+            @JsonProperty(value = "successful") Boolean successful,
+            @JsonProperty(value = "vot") Vot vot,
+            @JsonProperty(value = "isUkIssued") Boolean isUkIssued,
+            @JsonProperty(value = "age") Integer age) {
```

Comment:
I'm a bit sceptical that we need these annotations, but happy to leave if there's any uncertainty.

(TBH this whole thing can probably just be a record class...)

Reply:
Yeah, i intentionally left them, mostly out of fear

Reply:
You've twisted my arm and I've gone the whole hog